mcsema/Arch/ABI.cpp

```diff
@@ -518,6 +545,36 @@ static const char *GetVarImpl(
   return nullptr;
 }
 
+static void StorePackedType(llvm::BasicBlock *block, llvm::Value *ret_val) {
```

Comment:
Lets rename this to `TryStorePackedType`, returning a `bool`. Then the initial `CHECK` instead becomes a non-fatal condition and leads to a return of `false`.